File: tickets-agente-frontend/src/app/feature/sugerencia/sugerencia.ts
import { Rol } from "../rol/rol"

export interface Sugerencia{
  sugerenciaId: number,
  motivoId: number,
  sugerenci: string,
  created: Date,
  updated: Date,
  enable: boolean,
  roles: Rol[],
  motivoNombre: string
}

File: tickets-agente-backend/src/main/java/yavirac/tickets/feature/sugerencia/Sugerencia.java
package yavirac.tickets.feature.sugerencia;

import java.sql.Timestamp;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

import lombok.Data;

@Data
@Table("sugerencia")
public class Sugerencia {

    @Id
    @Column("sugerencia_id")
    private long sugerenciaId;
    
    private long motivoId;
    private String sugerenci;
    private Timestamp created;
    private Timestamp updated;
    private boolean enable;
    private boolean archive;
    
}
